.dockerignore

```diff
@@ -0,0 +1,34 @@
+# Include any files or directories that you don't want to be copied to your
```

Comment:
I did not realise this file was included as well sorry, I'd prefer to leave it in as it won't affect anything and saves me time but if you want it removed just say